challenges/objects-arrays.js

```diff
@@ -79,9 +116,18 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 {"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
-let lowerCase = [];
+
+
+let lowerCase = zooAnimals.map((animal, index) => {
+  return zooAnimals[index]['animal_name'].toLowerCase();
+});
```

Comment:
I think this would work, but map automatically indexes so you wouldn't necessarily need a separate index. 

ex:

```js

let lowercase = [];
lowercase = zooAnimals.map((animal) => {
    return animal.animal_name.toLowerCase();
});
return lowercase;


```